prow/cmd/generic-autobumper/main.go

```diff
@@ -476,14 +484,18 @@ func getVersionsAndCheckConsistency(prefixes []prefix, images map[string]string)
 	versions := map[string][]string{}
 	consistencyChecker := map[string]string{}
 	for _, prefix := range prefixes {
+		exceptions := sets.NewString(prefix.ConsistentImageExceptions...)
 		for k, v := range images {
 			if strings.HasPrefix(k, prefix.Prefix) {
+				image := imageFromName(k)
 				found, ok := consistencyChecker[prefix.Prefix]
-				if ok && (found != v) && prefix.ConsistentImages {
-					return nil, fmt.Errorf("%s:%s not bumped consistently for prefix %s(%s), expected version: %s", k, v, prefix.Prefix, prefix.Name, found)
-				} else if !ok {
+				// If consistency should be checked and this image is in ConsistentImageExceptions, record it.
+				if !ok && prefix.ConsistentImages && !exceptions.Has(image) {
```

Comment:
Hmm... you already do a check that `prefix.ConsistentImages` is `true` if `exceptions` is not empty during validation. So I think you could drop the `prefix.ConsistentImages` from this, no?

Reply:
You're right, but I think keeping it is cheap, and it's clearer what the expectations for this function are, without needing to cross-check the options validation.